Web/starter_project_barbies/components/Home-page/ServicesCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { ServiceDescription } from '../../interface/interfaces'
+
+function ServicesCard({data}:{data:ServiceDescription}) {
```

Comment:
reminder: i still don't know why we need to call the prop `data`
it should be descriptive and match with the type of that prop (`serviceDescription` in this case)

you can also remember to destruct it like i mentioned above

Reply:
[nit] and i think `ServiceCard` or `ServiceDescriptionCard` might be a better name for the component as it only has one service in it (not plural)